projects/angular/src/data/datagrid/built-in/filters/datagrid-numeric-filter.ts

```diff
@@ -22,28 +22,37 @@ import { DatagridNumericFilterImpl } from './datagrid-numeric-filter-impl';
   selector: 'clr-dg-numeric-filter',
   providers: [{ provide: CustomFilter, useExisting: DatagridNumericFilter }],
   template: `
-    <clr-dg-filter [clrDgFilter]="registered" [(clrDgFilterOpen)]="open">
-      <input
-        class="datagrid-numeric-filter-input"
-        #input_low
-        type="number"
-        autocomplete="off"
-        name="low"
-        [(ngModel)]="low"
-        [placeholder]="minPlaceholderValue"
-        [attr.aria-label]="minPlaceholderValue"
-      />
-      <span class="datagrid-filter-input-spacer"></span>
-      <input
-        class="datagrid-numeric-filter-input"
-        #input_high
-        type="number"
-        autocomplete="off"
-        name="high"
-        [(ngModel)]="high"
-        [placeholder]="maxPlaceholderValue"
-        [attr.aria-label]="maxPlaceholderValue"
-      />
+    <clr-dg-filter [clrDgFilter]="registered" [(clrDgFilterOpen)]="open" class="clr-form-horizontal">
+      <div class="datagrid-numeric-filter-input-wrapper">
```

Comment:
```suggestion
    <clr-dg-filter [clrDgFilter]="registered" [(clrDgFilterOpen)]="open">
      <div class="clr-form-horizontal datagrid-numeric-filter-form-wrapper">
```

I think it's better to the put the form layout class on the wrapper class. Also, I think the wrapper class should be renamed because it wraps a form, not a single input.